save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/HistoryView.kt

```diff
@@ -32,10 +34,15 @@ import react.dom.html.ReactHTML.div
 import react.dom.html.ReactHTML.td
 import react.table.columns
 
+import kotlin.js.Date
```

Comment:
We can use `kotlinx.datetime.LocalDateTime` to have a more universal solution among platforms

Reply:
Yes, I agree. But React elements require kotlin.js.Date